### PR TITLE
Steve/add video to home page

### PR DESCRIPTION
**Pull request by Steve-B-Murphy**

### CODE REVIEW

**Comment by Steve-B-Murphy** [comment]
@tristanlee85 Review code changes in index.js.
@tristanlee85 Please accept all changes in production.md.

File: pages/index.js [hunk 1]
```diff
@@ -128,6 +129,47 @@ const Home = ({ navData }) => {
...
+                  variant="contained"
+                  size="large"                                    
+                  className={classes.limeButtonContained}
+                >
+              Provide Feedback
+              </Button>
+            </Grid>
+              <Grid item md={4} sm={6} xs={12} className={classes.tutorial}>
+                <Button
+                    href="https://get.layer0.co/request-a-demo/?sgId=470c4986-7644-4416-b4ae-d7cad5814b59"
```
Comment (tristanlee85) [inline pages/index.js:158]:
```suggestion
                    href="https://get.layer0.co/request-a-demo/"
```

**Review by tristanlee85 — DISMISSED** [review]

File: guides/production.md [hunk 2]
```diff
@@ -66,12 +73,12 @@ dig <your-sub-domain>
 dig www.mywebsite.xyz
 
 # Result
-www.mywebsite.xyz.   599    IN    CNAME    d12ea738-71b3-25e8-c771-6fdd3f6bd8ba.layer0-limelight.link.
+www.mywebsite.xyz.   599    IN    CNAME    d12ea738-71b3-25e8-c771-6fdd3f6bd8ba.layer0.limelight.link.
```
Comment (tristanlee85) [inline guides/production.md:76]:
```suggestion
www.mywebsite.xyz.   599    IN    CNAME    d12ea738-71b3-25e8-c771-6fdd3f6bd8ba.layer0-limelight.link.
```

File: guides/production.md [hunk 3]
```diff
@@ -66,12 +73,12 @@ dig <your-sub-domain>
...
 ```
 
 #### Using an Apex Domain (e.g. mywebsite.xyz)
 
-To host your site on the apex domain, create multiple `A` records on your apex domain, with the following Anycast IP address values: 208.69.180.11, 208.69.180.12, 208.69.180.13, 208.69.180.14
+To host your site on the apex domain, create multiple `A` records on your apex domain, with the following Anycast IP address values: 208.69.180.11, 208.69.180.12, 208.69.180.13, 208.69.180.14.
```
Comment (tristanlee85) [inline guides/production.md:81]:
```suggestion
To host your site on the apex domain, create multiple `A` records on your apex domain, with the following Anycast IP address values: 208.69.180.11, 208.69.180.12, 208.69.180.13, 208.69.180.14
```

**Review by tristanlee85 — APPROVED** [review]

**Pull request merged**